<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2184</width>
    <height>984</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MKUltra</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="poly_2D_widget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>70</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="surface_loader_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>910</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load surface</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slice_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>850</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer_count_box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>590</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="surface_selector_box">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>860</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Level count:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="inner_shell_count_box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>690</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Inner shell count:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="inner_shell_distance_box">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>690</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>690</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Inner shell distance:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="infill_space_between_box">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>590</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>600</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Infill space between:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="infill_number_box">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>780</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Infill number (rot):</string>
    </property>
   </widget>
   <widget class="poly_3D_widget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="slice_bar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>920</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="support_checkbox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>920</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Show support</string>
    </property>
   </widget>
   <widget class="QPushButton" name="support_gen_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>850</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate support</string>
    </property>
   </widget>
   <widget class="mesh_3D_widget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="zoom_scrollbar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>801</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="support_space_between_box">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>770</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>780</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Support space between:</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>30</y>
      <width>801</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>poly_2D_widget</class>
   <extends>QWidget</extends>
   <header>poly_2D_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>poly_3D_widget</class>
   <extends>QWidget</extends>
   <header>poly_3D_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mesh_3D_widget</class>
   <extends>QWidget</extends>
   <header location="global">mesh_3D_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
